Update exchange-guide.mdx (#1056)
iotaledger/iota-wiki: next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

The `account.send()` function returns a `transaction` with its id. The `blockId` can be used later for checking the confirmation status. You can obtain individual transactions related to the given account using the `account.transactions()` function.

> `account.sendWithParams()` can be used to send to multiple addresses in the same transaction.

Author: ```suggestion
`account.sendWithParams()` can be used to send to multiple addresses in a single transaction.
```
